### DISCUSSION TITLE
Can someone sync the language .ts files to the latest version?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#1819] [General]
Hello, some languages lag behind the recent releases, including the German translation for the plus version.

I would like to translate the new and changed strings into German, but I need an updated .ts file for the German languages, but the remaining should not be left behind as well.

Thank you for providing the .ts files.

**Comment by isaak654** [comment]
A long-term solution would be to involve any contributor in the creation of the new GitHub worflow suggested in https://github.com/sandboxie-plus/Sandboxie/issues/1595 
Not only for Sandboxie Plus translations updates located [here](https://github.com/sandboxie-plus/Sandboxie/tree/master/SandboxiePlus/SandMan), but also for the new Classic strings which are added in [Sbie-English-1033.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/Sbie-English-1033.txt), so that they can be propagated in the other language files [here](https://github.com/sandboxie-plus/Sandboxie/tree/master/Sandboxie/msgs) marked as `Text-*-*.txt`

**Reply by bastik-1001** [reply]
Yes, for the long-term, sure. The reason I started this as a discussion, rather than a ticket, was that I don't wanted to see it as an issue, just that I needed an updated version for the .ts file. For the issue at hand, there is already a ticket.

Translating more than just one file, in particular if not all of them are translated with the same tool, seems to make a translation service more plausible and preferable.

**Reply by isaak654** [reply]
Meanwhile, I performed a manual update of all supported .ts files in the following commit: https://github.com/sandboxie-plus/Sandboxie/commit/168067470ce511f7ca6663869feb2a0403acbb05

**Reply by bastik-1001** [reply]
Thank you for updating these, so I could translate the new strings.

**Reply by isaak654** [reply]
A new lupdate workflow was merged in https://github.com/sandboxie-plus/Sandboxie/commit/81206b141307000a40045bdbca55382514d15a77
